test/mixbench/marshal.txt

```diff
@@ -1,73 +1,73 @@
-goos: darwin
+goos: linux
```

Comment:
Maybe we update this as part of another pull request, which includes the update of bench.md

Reply:
Sorry, I missed this. My intention wasn't to update the benches.